### PR TITLE
[NETBEANS-417] Adjust C file to handle new registry layout starting with jdk9 

### PR DESCRIPTION
**Pull request by ebarboni**

### CODE REVIEW

File: o.n.bootstrap/launcher/windows/jvmlauncher.cpp [hunk 1]
```diff
@@ -405,6 +407,13 @@ bool JvmLauncher::findJava(const char *minJavaVersion) {
     if (findJava(JRE_KEY, "", minJavaVersion)) {
         return true;
     }
+// scan for registry for jdk/jre version 9
+    if (findJava(JDK_POST9_KEY, "", minJavaVersion)) {
+        return true;
+    }
+    if (findJava(JRE_POST9_KEY, "", minJavaVersion)) {
+        return true;
+    }
```
Comment (nigjo) [inline o.n.bootstrap/launcher/windows/jvmlauncher.cpp]: Wouldn't it be better to check the new layout first to start the new version in favor of the old one?

Comment (ebarboni) [reply o.n.bootstrap/launcher/windows/jvmlauncher.cpp]: Adjusted to take more recent JDK first

**Comment by cowwoc** [comment]
Looks good to me.

**Comment by jlahoda** [comment]
Any objections against merging this? (We need to work on getting this into the binary .exe/.dll though.)

**Comment by geertjanw** [comment]
OK let’s merge.

**Pull request merged**